q2_emperor/_plot.py

```diff
@@ -79,8 +79,15 @@ def biplot(output_dir: str, biplot: skbio.OrdinationResults,
            sample_metadata: qiime2.Metadata, feature_metadata:
            qiime2.Metadata = None,
            ignore_missing_samples: bool = False,
+           invert: bool = False,
            number_of_features: int = 5) -> None:
 
+    if invert:
+        features = biplot.features.copy()
+        samples = biplot.samples.copy()
+        biplot.samples = features
+        biplot.features = samples
```

Comment:
@ElDeveloper like this?
```suggestion
        biplot.samples, biplot.features = samples, features
```

Reply:
I think you can even bypass the intermediate assignment to samples and features.